### ISSUE TITLE
Generate current task picture png

### ISSUE THREAD

**Issue by 219liangyi** [Shuffle/Shuffle#855]
In my third-party system, I hope to see the progress of project implementation. A function similar to activiti. In this way, I don't need everyone to log in to the shuffle background to view it
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/19781251/168479349-3429d819-93bc-47e5-a73e-4d858abbd046.png">

**Comment by frikky** [comment]
How do you want access to it? Right now, the image for the workflow is accessible through the API as a base64 blob. In Shuffle itself, this is rendered as you can see here: 

![image](https://user-images.githubusercontent.com/5719530/168597774-3f4347f3-1091-417e-91ae-c18b27aa5ef8.png)

Does this solve the problem?

**Comment by 219liangyi** [comment]
No, I need to let my users see where his work order has been executed. Instead of just seeing what the whole workflow looks like.
<img width="814" alt="image" src="https://user-images.githubusercontent.com/19781251/168764043-8db689c5-4206-4c5b-8acf-94fefcb422ab.png">
For example, in this figure, you can see the execution step of the work order. I need to let users see it in the third-party system
The execution progress of each work order is different. To show users the progress picture of a work order
